docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,275 @@
+
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a Teleport role with the `aws_role_arns` field
+listing the IAM role ARN created in the previous step. Create a file called
+`aws-dynamodb-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: aws-dynamodb-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_users:
+    - '*'
+```
+
+Create the new role:
+
+```code
+$ tctl create -f aws-dynamodb-access.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="aws-dynamodb-access"!)
+
+## Step 3/4. Install the Teleport Database Service
+
+Create an EC2 instance to host the Teleport Database Service, and attach the
+`TeleportDatabaseService` AWS IAM role to it. If you're hosting the service another
+way, you must provide AWS credentials to the service - see [AWS credentials
+configuration](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html)
+for more details.
+
+<Admonition type="warning" title="non-standard AWS regions">
+For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
+regions, please set the corresponding region in the `AWS_REGION` environment
+variable or in the AWS credentials file so that the Application Service can use
+the correct STS endpoint.
+</Admonition>
+
+### Generate a token
+
+A join token is required to authorize a Teleport Database Service instance
+to join the cluster. Generate a short-lived join token and save the output of
+the command:
+
+```code
+$ tctl tokens add \
+    --type=db \
+    --db-name=example-dynamodb
+    --db-protocol=dynamodb
+```
+
+Use the token provided by the output of this command in the next step.
+
+### Install and start Teleport
+
+Install Teleport on the host where you will run the Teleport Application
+Service. See our [Installation](../../installation.mdx) page for options
+besides Linux servers.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Edit `/etc/teleport.yaml` to configure the database entry:
+
+```yaml
+version: v3
+teleport:
+  nodename: CHANGEME
+  data_dir: /var/lib/teleport
+  proxy_server: teleport.example.com
+  auth_token: (=presets.tokens.first=)
+  ca_pin:
+  - (=presets.ca_pin=)
+db_service:
+  enabled: "yes"
+  # Matchers for database resources created with "tctl create" command.
+  # For more information: https://goteleport.com/docs/database-access/guides/dynamic-registration/
+  resources:
+  - labels:
+      "*": "*"
+  # Lists statically registered databases proxied by this agent.
+  databases:
+    - name: "example-dynamodb"
+      protocol: "dynamodb"
+      # optional uri, if uri is set then AWS region can be extracted from it
+      # or if AWS region is already set then the regions must match.
+      # uri: "dynamodb.us-east-1.amazonaws.com:443"
+      static_labels:
+        env: "dev"
+      aws:
+        region: "us-east-1"
+        account_id: "(=aws.aws_access_key=)"
```

Comment:
same here. a example of  AWS AccountID value should be used here instead of aws access key.

Reply:
[Same response](https://github.com/gravitational/teleport/pull/19956/files#r1065985779).